### PR TITLE
Better ping command

### PR DESCRIPTION
**Pull request by TheKodeToad**

### CODE REVIEW

File: src/commands/general/ping.rs [hunk 1]
```diff
@@ -1,11 +1,34 @@
+use std::time::{Duration, Instant};
+
```
Comment (getchoo) [inline src/commands/general/ping.rs:2]:
```suggestion
```

File: src/commands/general/ping.rs [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
 use crate::{Context, Error};
 
 use log::trace;
+use poise::CreateReply;
+
+const PING_PREFIX: &str = "<:catstareback:1078622789885497414> Pong!";
```
Comment (getchoo) [inline src/commands/general/ping.rs:8]:
```suggestion
```

File: src/commands/general/ping.rs [hunk 3]
```diff
@@ -1,11 +1,34 @@
...
+
+	response
+		.edit(
+			ctx,
+			CreateReply::default().content(format!(
+				"{PING_PREFIX}\n\nRTT: {rtt}ms\nGateway: {gateway_ping}",
+			)),
+		)
+		.await?;
+
```
Comment (getchoo) [inline src/commands/general/ping.rs:32]: this is how it was implemented before. i don't see many cases where this is extremely useful in the first place, let alone RTT as well; so i don't see a reason to "improve" on the previous implementation either
```suggestion
  let resp = format!("Pong! (`{}ms`)", ctx.ping().await.as_millis());
  ctx.say(resp).await?;
```

**Review by getchoo — CHANGES_REQUESTED** [review]

**Comment by TheKodeToad** [comment]
This is what most bots have so i don't see why not?

**Review by Scrumplex — APPROVED** [review]
IMO this is good

**Pull request merged**